app/controllers/thredded/messageboards_controller.rb

```diff
@@ -44,6 +44,13 @@ def update
       end
     end
 
+    def destroy
+      @messageboard = Thredded::Messageboard.friendly_find!(params[:id])
+      authorize @messageboard, :destroy?
+      @messageboard.destroy!
+      redirect_to root_path, notice: 'Messageboard deleted'
```

Comment:
Please internationalize all the UI strings.

It's OK to use machine translation for languages you don't speak.
Once you've added the English strings to `config/locales/en.yml`, run:

```bash
GOOGLE_TRANSLATE_API_KEY="your api key" bundle exec i18n-tasks translate-missing
```

Reply:
Done!